web/client/epics/wfsquery.js

```diff
@@ -52,7 +52,9 @@ const extractInfo = (data) => {
 };
 
 const featureTypeSelectedEpic = action$ =>
-    action$.ofType(FEATURE_TYPE_SELECTED).switchMap(action => {
+    action$.ofType(FEATURE_TYPE_SELECTED)
+    .filter(action => action.url !== null)  // Axios will try to request a /null url without this filter
```

Comment:
I think we should NOT launch the action with a null url, if it doesn't make sense